test/src/DumpVario.cpp

```diff
@@ -28,7 +28,11 @@ Copyright_License {
 
 #include <stdio.h>
 #include <stdlib.h>
-#include <unistd.h>
+#if defined(_MSC_VER)
+# include <windows.h>
```

Comment:
> the function declarations are in windows.h

MSDN documentation disagrees.

Reply:
Yes, in `windows.h` fom MinGW - but not in `Windows.h` from
`C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um` - used from MSVC compiler
From my point of view MSDN is right about that.

Reply:
I have **no idea** what your response means. I don't understand any of it.

Reply:
With MSVC I have no unistd.h available, this is a part of the MinGW package only available for the GCC compiler. With a pure Windows system with VisualeStudio installed there is no header file unistd.h! 
All the Windows system headers you can find in the current Windows Kit folder.

Reply:
I know you have no unistd.h, but ... what's your point?
My point was that your statement about windows.h is wrong. What do your replies have to do with that? You confuse me.